server/apis/v1/handler.go

```diff
@@ -99,25 +99,6 @@ func (h *handler) GetPipeline(c *gin.Context) {
 	c.JSON(http.StatusOK, pl)
 }
 
-// ListNamespaces is used to provide all the namespaces that have numaflow pipelines running
```

Comment:
For cluster scoped, can we use this for type-ahead search?

Reply:
It can be.

However the type-ahead experience might be conflicted with existing dropdown from cookie, it needs some effort to figure out the best user experience if we want to support that. We can bring it back when we implement it.